### PR TITLE
Feature: Assignment Listing and Creation

### LINKED ISSUE #86: Change all instances of is_staff? to is_current_staff?

### PR DESCRIPTION
**Pull request by chenhoward**
## About
Allows us to create assignments and for students to view the list of assignments and due dates.

Fixes #86 and #150 

## Notes
It contains the migrations for submission but none of the models/controllers/views.  We can only have integer points to keep things chill.

### CODE REVIEW

File: app/controllers/assignments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
+class AssignmentsController <  ApplicationController
```
Comment (warrenshen) [inline app/controllers/assignments_controller.rb]: Is it just me or are there two spaces before the `ApplicationController`?

File: app/views/assignments/index.html.slim [hunk 2]
```diff
@@ -0,0 +1,14 @@
+.container
+  h1 Assignements
```
Comment (warrenshen) [inline app/views/assignments/index.html.slim]: Typo.

**Comment by chenhoward** [comment]
Ready for round 2 @warrenshen

**Comment by warrenshen** [comment]
The rest looked good @chenhoward, merging it in.

**Pull request merged**